bridges/testing/tests/0004-asset-transfer-over-ah/run.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -e
+
+source "${BASH_SOURCE%/*}/../../framework/utils/common.sh"
+source "${BASH_SOURCE%/*}/../../framework/utils/zombienet.sh"
+
+export ENV_PATH=`realpath ${BASH_SOURCE%/*}/../../environments/rococo-westend`
+
+$ENV_PATH/spawn.sh --init --start-relayer --permlanes &
+env_pid=$!
+
+ensure_process_file $env_pid $TEST_DIR/rococo.env 600
+rococo_dir=`cat $TEST_DIR/rococo.env`
+echo
+
+ensure_process_file $env_pid $TEST_DIR/westend.env 300
+westend_dir=`cat $TEST_DIR/westend.env`
+echo
+
```

Comment:
I would add here opening bridges as a separate step - e.g.

```
run_zndsl ${BASH_SOURCE%/*}/ahr-open_bridge.zndsl $rococo_dir
run_zndsl ${BASH_SOURCE%/*}/ahw-open_bridge.zndsl $rococo_dir
```
and those zdnsl will trigger those scripts: `open-bridge-lane-westend-rococo-local` / `open-bridge-lane-rococo-westend-local`


These is better, because later we will add penpal and opening bridge from penpal, and I don't want to have this bridge opening as a part of startup script. After startup we could have multiple scenarios for opening different bridges...